internal/controller/operatorconfigmap_controller.go

```diff
@@ -241,6 +221,26 @@ func (c *OperatorConfigMapReconciler) Reconcile(ctx context.Context, req ctrl.Re
 			c.log.Error(err, "failed to perform precheck for deploying CSI")
 			return ctrl.Result{}, err
 		} else if deployCSI {
+			if err := c.reconcileWebhookService(); err != nil {
```

Comment:
should we move https://github.com/red-hat-storage/ocs-client-operator/blob/5e08b7357256da96ddb2448df9c81766ced21ed7/internal/controller/operatorconfigmap_controller.go#L215-L218 inside deployCSI as well? is there any case we cant to createConsolePlugin when we are not managing/deploying CSI?

Reply:
it isn't moved as that doesn't depend on CSI however console deployment is dependent on this reconcile result and so not moved.

Reply:
okay let me rephrase my question sorry for not being clear, do we need to deploy consolePlugin if we are not managing csi? i know both are not dependent but do we have any case like this?

Reply:
ack, I'll be reworking on this PR to shutdown unless explicitly asked to run as that should be the correct fix.

Reply:
done.